.travis.yml

```diff
@@ -51,10 +51,18 @@ script:
       ./build_docs.sh;
     fi;
 
+ # Check repo size does not expand too much
+    - if [ "${TEST}" == "extra" ]; then
```

Comment:
In pyqtgraph, a `TEST` environmental variable is defined.  We are not defining such an environmental variable but we could use the `BUILD_DOCS` environmental variable as proxy. Change this line to be 

`- if [ "$BUILD_DOCS" == "true" ]; then`